robottelo/host_helpers/satellite_mixins.py

```diff
@@ -4,12 +4,15 @@
 
 import requests
 
+from robottelo import ssh
```

Comment:
You should not need this as you can directly access `put` as a satellite object method inherited from ContentHost class via Capsule class.

Reply:
Updated